### PR TITLE
Formally deprecate Block and ScrollableViewport

### PR DESCRIPTION
**Pull request by abarth**
The replacements are ListView and SingleChildScrollView, respectively.

### CODE REVIEW

**Comment by abarth** [comment]
@Hixie

File: packages/flutter/test/material/refresh_indicator_test.dart [hunk 1]
```diff
@@ -28,7 +28,7 @@ void main() {
       new RefreshIndicator(
         scrollableKey: scrollableKey,
         refresh: refresh,
-        child: new Block(
+        child: new Block( // ignore: DEPRECATED_MEMBER_USE
```
Comment (Hixie) [inline packages/flutter/test/material/refresh_indicator_test.dart]: conflicts ahoy! :-)

**Comment by Hixie** [comment]
LGTM

File: packages/flutter/test/material/refresh_indicator_test.dart (hunk 1 above)
Comment (abarth) [inline packages/flutter/test/material/refresh_indicator_test.dart]: Sorry.  :(

You can just chuck my version when you hit conflicts.

Comment (Hixie) [inline packages/flutter/test/material/refresh_indicator_test.dart]: No worries, they'll be trivial.

**Pull request merged**